pkg/query-service/app/logs/v3/query_builder.go

```diff
@@ -330,17 +307,30 @@ func orderBy(items []v3.OrderBy, tags []string) string {
 	for _, item := range items {
 		if item.ColumnName == constants.SigNozOrderByValue {
 			orderBy = append(orderBy, fmt.Sprintf("value %s", item.Order))
+			addedToGroupBy[item.ColumnName] = true
+		}
+	}
+
+	// add the remaining items
+	if panelType == v3.PanelTypeList {
+		for _, item := range items {
+			// since these are not present in tags we will have to select them correctly
+			if !addedToGroupBy[item.ColumnName] {
```

Comment:
The list type won't have the groupby since we show the raw data. But I don't see that as a problem here since `addedToGroupBy` will be empty. You can may be write some comments for the reader who goes through this code.

Reply:
makes sense